backend/tournesol/migrations/0021_fill_new_entity_fields.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.2.11 on 2022-02-03 14:08
+
+from django.db import migrations
+
+
+ENTITY_UID_YT_NAMESPACE = "yt"
+ENTITY_TYPE_VIDEO = "video"
+
+
+def migrate_forward(apps, schema_editor):
+    """
+    Set the `uid` and the `type` field of all entities present in the
+    database (supposedly only YT videos are present).
+    """
+    Entity = apps.get_model("tournesol", "Entity")
+    for entity in Entity.objects.iterator():
+        entity.uid = "{}:{}".format(ENTITY_UID_YT_NAMESPACE, entity.video_id)
+        entity.type = ENTITY_TYPE_VIDEO
+        entity.save()
```

Comment:
"update_fields" can be used here, to avoid potential conflicting updates during the migration (and slightly better performance).
```suggestion
        entity.save(update_fields=["uid", "type"])
```